QA_Robot_Practice/utils/RandomWordsUtil.robot

```diff
@@ -0,0 +1,8 @@
+*** Settings ***
+Library    GenerateWordsLibrary.py
+
+*** Keywords ***
+Random credential
+    [Arguments]  ${wordLength}
```

Comment:
Documentation is required. Default and return values should be desctibed in addition to the general description